### ISSUE TITLE
RequestAFter corrected to RequestAAfter

### ISSUE THREAD

**Issue by SOF3** [crate-ci/typos#743]
```console
$ typos test.txt --diff
--- test.txt    original
+++ test.txt    fixed
@@ -1 +1 @@
-RequestAFter
+RequestAAfter
```

It should be corrected to `RequestAfter` instead.

**Comment by epage** [comment]
Any thoughts on *how* we can identify invalid wordsplits like this?

**Comment by SOF3** [comment]
Citation needed, but my understanding is that single-letter words (especially articles like "a") are very uncommon in camel/pascal case identifiers. If a word is detected to be missing its first character and the previous character is uppercase, could we consider merging them/

**Comment by epage** [comment]
> Citation needed, but my understanding is that single-letter words (especially articles like "a") are very uncommon in camel/pascal case identifiers.

I agree and with the fact that this can bite us in other ways, and the complexity involved in trying to support it, I would lean towards us not supporting this.
